data/templates/nginx/plain/yunohost_admin.conf

```diff
@@ -65,10 +65,7 @@ server {
 
     location /yunohost {
         # Block crawlers bot
-        if ($http_user_agent ~ (crawl|Googlebot|Slurp|spider|bingbot|tracker|click|parser|spider|facebookexternalhit) ) {
-            return 403;
```

Comment:
Reading the doc, I guess I'm okay with the X-Robots-Tag. Nevertheless I don't understand why we should remove this line ? Sure it probably does not cover all user agents, but yet : 
- we could improve this list
- I am more confident about this approach that simply tells "fuck off" to the bots as it better covers the existence of the /yunohost subpath in the first place than a simple header.

So imho those are complementary approach.

Reply:
mh... ok. True. I can readd this line (I don't remember why I removed it).

Reply:
done